data/xapleaux/xapleaux ships.txt

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) 2021 by MasterOfGrey
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+ship Ascomy
+	sprite ship/ascomy
+	thumbnail thumbnail/ascomy
+	uncapturable
```

Comment:
What is it about Xapleaux ships that makes them uncapturable? 
The Xapleaux themselves seem to favour broadly similar conditions to the player (otherwise you wouldn't be able to survive on their planets), so it's presumably not the case that the internal conditions are unsurvivable for humans.

Reply:
The fact that they're currently unbalanced hulks of temporary stats. ;)